uelc/main/models.py

```diff
@@ -444,12 +444,13 @@ def is_curveball(self, current_section, pageblocks=None):
         block = None
         if pageblocks is None and current_section is not None:
             pageblocks = current_section.pageblock_set.all()
-        if pageblocks:
```

Comment:
I'm concerned that removing this if statement would raise: `TypeError: 'NoneType' object is not iterable` if both `pageblocks` and `current_section` is None. I'm adding unit tests to this method, since it's missing tests.